src/components/Button/Button.module.scss

```diff
@@ -74,7 +74,9 @@ $disabled-on-primary-color-opacity: 0.5;
 }
 
 .button:active:not(.preventClickAnimation) {
-  transform: scale(0.95) translate3d(0, 0, 0);
+  --button-scale-factor: 0.95;
+  transform: scale(var(--button-scale-factor)) translate3d(0, 0, 0);
+  margin: calc(1% * (1 - var(--button-scale-factor)));
```

Comment:
This makes other elements on the screen to be shifted when the button is pressed:

https://github.com/mondaycom/monday-ui-react-core/assets/18269880/01f4cf90-1d55-4aa7-8d5d-d7e961b8cb70

Reply:
I will check the issue again.
Do you have a visual testing tool, such as Applitool or Percy?

Reply:
Yes, we use Chromatic

Reply:
There are two potential solutions:

- adding a DOM element as a wrapper
- replace the scale reduction with box shadow

I don't like both changes because they change the original design or impact performance.

Reply:
I completely agree, it's a tricky one 💭